src/main/scala/com/microsoft/azure/iot/kafka/connect/sink/C2DMessage.scala

```diff
@@ -0,0 +1,9 @@
+/*
+ * // Copyright (c) Microsoft. All rights reserved.
+ */
+
+package com.microsoft.azure.iot.kafka.connect.sink
+
+import java.util.Date
+
+case class C2DMessage(messageId: String, message: String, deviceId: String, expiryTime: Option[Date] = None)
```

Comment:
we could support the ACK modes, see com.microsoft.azure.sdk.iot.service.sdk.DeliveryAcknowledgement: None, NegativeOnly, PositiveOnly, Full